mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -69,6 +74,15 @@ async function interpretPartialBeginToken(token: LexerToken) : Promise<string> {
             const partial = getPartialInCodeBlock(json);
             return partial;
         } 
+        
+        if (config.hasOwnProperty("transformation")) {
```

Comment:
Simply:
`if (config.transformation)`

Reply:
You're right!

Should be: `if (config.transformation !== undefined)`

Using `hasOwnProperty` does not type check the value you are passing to the method (could be `hasOwnProperty("transformationWithTypo")` and you wouldn't get a warning).